paddle/fluid/framework/tensor.cc

```diff
@@ -40,7 +40,9 @@ void* Tensor::mutable_data(platform::Place place, proto::VarType::Type type,
   PADDLE_ENFORCE_GE(numel(), 0,
                     "When calling this method, the Tensor's numel must be "
                     "equal or larger than zero. "
-                    "Please check Tensor::Resize has been called first.");
+                    "Please check Tensor::Resize has been called first. "
```

Comment:
please check Tesnor dims, InferShape or Resize

Reply:
Discussed with @phlrain, since `InferShape` concept is not defined in Tensor. We will change the error message to : 
```
Please check Tensor::dims(), or Tensor::Resize() has been called first.
```

Reply:
Done